src/rust/multi.comick/src/lib.rs

```diff
@@ -10,8 +10,8 @@ use aidoku::{
 pub mod helper;
 pub mod parser;
 
-const BASE_URL: &str = "https://comick.cc";
-const API_URL: &str = "https://api.comick.cc";
+const BASE_URL: &str = "https://comick.fun";
+const API_URL: &str = "https://api.comick.fun";
```

Comment:
why did you change back to the old `.fun` domain? `.cc` is the new one

Reply:
It is mentioned in API docs to use  `.fun` for api and `.cc` for website

![image](https://github.com/Skittyblock/aidoku-community-sources/assets/22360076/c40ef9ed-bf61-42a6-9644-37d6500dbf93)

Reply:
shouldn't `BASE_URL` be `.cc`?